src/main/java/pl/agasior/interviewprep/dto/GetQuestionsByTagsRequest.java

```diff
@@ -11,6 +12,8 @@
 @Getter
 @Value
 public class GetQuestionsByTagsRequest {
+
+    @JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
```

Comment:
Could you exaplain why is this needed?

Reply:
JSON parser in java, when got 1 object from front, didn't created it ass array and got runtimeException when parsing